Makefile.ide

```diff
@@ -95,8 +95,7 @@ ide-toploop: ide-$(BEST)loop
 ifeq ($(HASCOQIDE),opt)
 $(COQIDE): $(LINKIDEOPT)
 	$(SHOW)'OCAMLOPT -o $@'
-	$(HIDE)$(OCAMLOPT) $(COQIDEFLAGS) $(OPTFLAGS) -o $@ unix.cmxa threads.cmxa lablgtk.cmxa \
-		lablgtksourceview2.cmxa str.cmxa $(IDEFLAGS:.cma=.cmxa) $^
+	$(HIDE)$(OCAMLOPT) $(COQIDEFLAGS) $(OPTFLAGS) -package str,lablgtk2.sourceview2 -linkpkg $(IDEFLAGS:.cma=.cmxa) -o $@ $^
```

Comment:
Why this change?

Reply:
Otherwise linking will fail, as the base modules are compiled with bisect and need the bisect unit. Note that this change is already present on #6859 so things will work "magically" there.

ocamlfind does some magic when using `-linkpkg` so indeed if the `cma` depends on bisect it will be automatically added to close the object dependency chain.